src/app.conf

```diff
@@ -0,0 +1,50 @@
+server {
```

Comment:
what is the reason to have two nginx configuration files?

Reply:
you haven't ansered here.

Reply:
removed second nginx config file, this was an accident